### PR TITLE
Add bundle snapshots

### LINKED ISSUE #53: Add snapshot testing for bundles

### PR DESCRIPTION
**Pull request by chee**
closes #53

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,3 +77,23 @@ Options:
...
+
+## Development
+
+Scrumple is written in [rust](https://www.rust-lang.org/) :crab:
+
+### Updating snapshots
+
+Scrumple makes use of [insta](https://docs.rs/insta/0.16.0/insta/) for snapshots.
+
+If you have changes some code and `cargo test` is failing, you can check the new snapshot is correct (which is available at `src/test/$snapshot_name.snap.new`) and then replace the old one.
```
Comment (JakeChampion) [inline README.md]:
```suggestion
If you have changed some code and `cargo test` is failing, you can check the new snapshot is correct (which is available at `src/test/$snapshot_name.snap.new`) and then replace the old one.
```

File: README.md [hunk 2]
```diff
@@ -16,6 +16,14 @@ Scrumple is based on a now-deleted tool called Pax, but we have added support fo
 
 This is a developer tool, not production software. It's designed to give the developer super-fast feedback when building components, not for building a production application.
 
+## Installation
```
Comment (JakeChampion) [inline README.md:19]:
```suggestion
## Installation

Scrumple requires `npm` to be installed.
```

Comment (JakeChampion) [reply README.md:19]: I think Scrumple requires npm, based on this line -- https://github.com/Financial-Times/scrumple/pull/54/files#file-src-main-rs-L71

Comment (chee) [reply README.md:19]: it doesn't. that's only for tests and benchmarks. and it isn't even actually required in that test.

File: README.md (hunk 1 above)
Comment (chee) [reply README.md]: thank you

**Comment by chee** [comment]
converting this to a draft because im working on something else that is taking all my cpu right now

**Comment by chee** [comment]
"\r\n" != "\n"

**Comment by chee** [comment]
woo

**Comment by chee** [comment]
it passed

**Review by JakeChampion — APPROVED** [review]

**Pull request merged**